workspaces/client/app/components/feature/user/profile-card.tsx

```diff
@@ -17,6 +18,7 @@ export const ProfileCard: React.FC<Props> = ({
 	roles,
 	bio,
 }) => {
+	const { reactContent } = useMarkdown(bio);
```

Comment:
`reactContent` だと今後増えたときにわかりづらいかも？ (ほかの部分も同様で)

```suggestion
	const { reactContent: bioPreviewContent } = useMarkdown(bio);
```
↑ 雑命名です